gui_qt/project_state.py

```
@@ -68,6 +68,113 @@ def get_latest_manifest_path(self) -> Path | None:
+        normalized_game_root = self._normalized_path_text(game_root)
+
+        # Prefer the CLI's current pointer so newly-created jobs are visible
+        # immediately even when the broader history index is cached.
+        latest = self.get_latest_manifest_path()
+        if latest is not None:
+            try:
+                manifest = self.load_manifest_file(latest)
+                if self._manifest_matches(manifest, expected_mode, normalized_game_root):
+                    return latest
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)